wormhole-connect/src/hooks/useRoutesQuotesBulk.ts

```diff
@@ -118,7 +122,12 @@ const useRoutesQuotesBulk = (routes: string[], params: Params): HookReturn => {
         acc[route] = quotes[index];
         return acc;
       }, {} as Record<string, QuoteResult | undefined>),
-    [routes.join(), quotes],
+    [
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+      routes.join(),
```

Comment:
Do we need `routes.join()` anymore since `routes` is in the array? My thought was `routes.join()` which returns a `string` was here so unnecessary re-renders aren't performed when the `routes` reference changes, but the actual content doesn't. However it looks like `routes` is memoized upstream so maybe this is a moot point now

Reply:
Great point about the upstream variable being memoized. I've actually tried this without join and it was still getting the infinite callbacks to the side-effect. I think React might be re-creating the array ref downstream.